bazel/packaging/packaging.bzl

```diff
@@ -18,17 +18,54 @@ def _is_versioned_so(file):
             return False
     return True
 
-def _impl(ctx):
+def _is_versioned_so(file):
+    return _is_versioned(file, "lib")
+
+def _is_dynamic_loader(file):
+    return _is_versioned(file, "ld")
+
+def _override_binary_rpath(ctx, path_override, original_binary):
+    patched_binary = ctx.actions.declare_file("{}_patched".format(original_binary.path))
+
+    ctx.actions.run(
+        inputs = [original_binary],
+        outputs = [patched_binary],
+        executable = ctx.executable._patchelf,
+        arguments = ["--force-rpath", "--set-rpath", path_override, original_binary.path, "--output", patched_binary.path],
```

Comment:
Why are we setting `--force-rpath`? https://man.archlinux.org/man/patchelf.1.en#force-rpath